### PR TITLE
executor: fix panic for point get executor during stale read

### PR DESCRIPTION
**Pull request by Yisaer**
### What problem does this PR solve?

```sql
create table test.table1 (id int primary key, a int);
explain analyze select * from test.table1 as of timestamp 'ts' where id = 1; (got panic)
```

What's Changed:

fix panic

### Check List 

Tests 

- Unit test

### Release note 

- No release note

### CODE REVIEW

File: executor/stale_txn_test.go [hunk 1]
```diff
@@ -733,6 +733,11 @@ func (s *testStaleTxnSuite) TestAsOfTimestampCompatibility(c *C) {
 		c.Assert(err.Error(), Matches, ".*as of timestamp can't be set in transaction.*")
 		tk.MustExec("commit")
 	}
+	_, err := tk.Exec(`create table test.table1 (id int primary key, a int);`)
+	c.Assert(err, IsNil)
+	time1 = time.Now()
+	_, err = tk.Exec(fmt.Sprintf("explain analyze select * from t5 as of timestamp '%s' where id = 1;", time1.Format("2006-1-2 15:04:05.000")))
```
Comment (xhebox) [inline executor/stale_txn_test.go:739]: You could just use `MustExec` and `CURRENT_TIMESTAMP(6)` instead of the additional format.

**Review by xhebox — COMMENTED** [review]
Verified locally, rest LGTM.
Maybe I should apply for exec reviewer...

File: executor/stale_txn_test.go (hunk 1 above)
Comment (Yisaer) [reply executor/stale_txn_test.go]: updated.

Comment (xhebox) [reply executor/stale_txn_test.go]: May we also add the corresponding `drop` statement?

**Review by xhebox — APPROVED** [review]

**Review by nolouch — APPROVED** [review]
lgtm

**Pull request merged**